compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -1401,7 +1425,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                         .unwrap_or(Ty::new_misc_error(self.tcx)),
                 );
 
-                for _matched_method in self.probe_for_name_many(
+                for _ in self.probe_for_name_many(
```

Comment:
I'd prefer leaving these underscore-first names unchanged, no need for the churn.